packages/core/src/types/index.ts

```diff
@@ -47,22 +47,22 @@ type NormalizedResolveTypeFn = (value: any, key: string) => NormalizedPropType;
 export function listOf(typeName: string) {
   return {
     name: `${typeName}List`,
-    properties: {},
+    properties: Object.create(null),
     items: typeName,
   };
 }
 
 export function mapOf(typeName: string) {
   return {
     name: `${typeName}Map`,
-    properties: {},
+    properties: Object.create(null),
     additionalProperties: () => typeName,
   };
 }
 
 export const SpecExtension: NormalizedNodeType = {
   name: 'SpecExtension',
-  properties: {},
+  properties: Object.create(null),
```

Comment:
We use `properties: {}` in lots of other node types. Don't we need to update them all?

Reply:
I just checked. I think simpler to check for own properties.